README.md

```diff
@@ -5,7 +5,6 @@
 The bitwarden help center project is written using Jekyll, which is hosted/deployed to a GitHub pages branch ([https://help.bitwarden.com/](https://help.bitwarden.com/)).
 
 # Build/Run
-
```

Comment:
can you please undo this change and add the new line back so that git history doesn't show these kind of changes.